Description: LED operation should not block S3/S4
Author: Adam Lee <adam8157@gmail.com>

Some Lenovo Thinkpad models don't have a suspend LED, this should not
block S3/S4.

--- pm/sleep.d/95led
+++ pm/sleep.d/95led
@@ -14,3 +14,5 @@
 	*) exit $NA
 		;;
 esac
+
+exit 0
